tests/bsa/plugin.cpp

```diff
@@ -77,19 +77,63 @@ TEST_CASE("find_archive_name", "[src]")
 
 TEST_CASE("remake dummy plugins")
 {
+    GIVEN("a game that can load limited archives per plugin, a directory with no plugins, and two archives")
+    {
+        auto dir = prepare_dir(
+            std::vector{u8"Test.bsa"sv, u8"Test1.bsa"sv});
+
+        WHEN("remake_dummy_plugins is called")
+        {
+            auto sets = Settings::get(btu::Game::SSE);
+            remake_dummy_plugins(dir.path(), sets);
+
+            THEN("two dummy plugins were created")
+            {
+                CHECK(list_plugins(dir.path(), sets).size() == 2);
+            }
+            THEN("the dummy plugins have the correct content")
+            {
+                auto dummy_content = require_expected(btu::common::read_file(dir.path() / u8"Test.esp"));
+                CHECK(dummy_content == sets.dummy_plugin.value());
+                dummy_content = require_expected(btu::common::read_file(dir.path() / u8"Test1.esp"));
+                CHECK(dummy_content == sets.dummy_plugin.value());
+            }
+        }
+    }
+    GIVEN("a game that can load unlimited archives per plugin, a directory with no plugins, and two archives")
+    {
+        auto dir = prepare_dir(std::vector{u8"Test.bsa"sv, u8"Test1.bsa"sv});
+
+        WHEN("remake_dummy_plugins is called")
+        {
+            auto sets = Settings::get(btu::Game::FNV);
+            remake_dummy_plugins(dir.path(), sets);
+
+            THEN("one dummy plugin was created")
+            {
+                CHECK(list_plugins(dir.path(), sets).size() == 1);
+            }
+            THEN("the dummy plugin has the correct content")

```

Comment:
nit: you could use AND_THEN (but it doesn't matter)

Reply:
Changed.